modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -124,6 +135,51 @@ public ObjectValidationRule addObjectValidationRule(
 		return objectValidationRule;
 	}
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public ObjectValidationRule addOrUpdateObjectValidationRule(
```

Comment:
Do we need this if we start using ObjectValidationResource in the Object Definition Resource?